Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/ApplyPaalmanPingsCorrection.py

```diff
@@ -65,7 +79,8 @@ def PyExec(self):
         prog_container.report('Starting algorithm')
 
         # Units should be wavelength
-        sample_unit = s_api.mtd[self._sample_ws_name].getAxis(0).getUnit().unitID()
+        sample_unit = \
+            s_api.mtd[self._sample_ws_name].getAxis(0).getUnit().unitID()
```

Comment:
This fails the 80 char limit on my `flake8` tool. Do we have a different standard in Mantid?